### PR TITLE
msglist: Update more elements closer to new, 2023+ design, to prepare for light/dark

### PR DESCRIPTION
**Pull request by chrisbobbe**
Like #657, but this is for the message list only (not content), and it completes my sweep through the message list. Screenshots coming soon.

### CODE REVIEW

**Comment by chrisbobbe** [comment]
| Before | After |
| --- | --- |
| D1B915D4-4D07-4157-8572-F95974A54586 | F6A4402C-1A8F-412F-B6D4-FDD738DAF3CC |
| A4D716F4-B5B5-4357-BAE6-55D6F4286BBF | 2F354A04-E246-4B47-914D-5D9E056E5F48 |

I don't have screenshots for the unsubscribed-stream icon color change, because (as discussed in the office just now) it doesn't seem possible to get to a message list that shows that icon.

**Comment by chrisbobbe** [comment]
Just pushed a small change (making the icon in the DM recipient header be the same color as the text) and updated the screenshots above, to show it.

File: lib/widgets/message_list.dart [hunk 1]
```diff
@@ -892,6 +889,7 @@ class MessageWithPossibleSender extends StatelessWidget {
 
   final MessageListMessageItem item;
 
+  // TODO(#95) unchanged in dark theme?
```
Comment (gnprice) [inline lib/widgets/message_list.dart:892]: Let's squash these three:

14c3f52d0 msglist [nfc]: Comment on _starColor about dark theme
a0ebda6bb msglist [nfc]: Comment on _kMessageTimestampColor about dark theme
61be55b75 msglist [nfc]: Comment with TODO(#95) on mark-as-read button

oh and this previous commit:
53440c7c9 msglist [nfc]: Add TODO on date-separator line about dark theme

They're one line each, all adding comments on the same theme, and none of them call for significant discussion in the commit message.

**Review by gnprice — COMMENTED** [review]
Thanks for cleaning these up, and the sleuthing on those date colors! Small comments below; otherwise looks good.

File: lib/widgets/message_list.dart [hunk 2]
```diff
@@ -519,8 +520,13 @@ class DateSeparator extends StatelessWidget {
 
   final Message message;
 
-  // This color matches recipient headers.  TODO(design) is that what we want?
-  static final _textColor = Colors.black.withOpacity(0.4);
+  // TODO(design) Do we want to match web more closely?
+  //   - width of 1 logical pixel instead of 1 physical pixel
+  //   - opacity 0.15
```
[the remark this answers is not in the document]
Comment (chrisbobbe) [reply lib/widgets/message_list.dart:525]: I see; yeah, that makes sense.

**Comment by chrisbobbe** [comment]
Thanks for the review! Revision pushed.

**Comment by gnprice** [comment]
Thanks! Looks good; merging.

**Pull request merged**